sqlalchemy_file/file.py

```diff
@@ -41,28 +42,38 @@ class File(BaseFile):
 
     def __init__(
         self,
-        content: Any,
+        content: Any = None,
+        content_path: Optional[str] = None,
         filename: Optional[str] = None,
         content_type: Optional[str] = None,
         **kwargs: Dict[str, Any],
     ) -> None:
```

Comment:
```suggestion
        filename: Optional[str] = None,
        content_type: Optional[str] = None,
        content_path: Optional[str] = None,
        **kwargs: Dict[str, Any],
    ) -> None:
```
 It might be better to move the `content_path` parameter down. This way, existing calls to the function won't break